influxdb3/tests/server/auth.rs

```diff
@@ -27,8 +27,9 @@ static COMMAND: Mutex<Option<DropCommand>> = parking_lot::const_mutex(None);
 
 #[tokio::test]
 async fn auth() {
-    const HASHED_TOKEN: &str = "784e3a542d6e29ff4b74548b24a9adb31752e15010a430dc214f7acf1a701cb9ec58bbc4b1bc6620c070e778144c8c9129d4e51e1bb5ee29f0f460fa002a1694";
-    const TOKEN: &str = "apiv3_1be71c8fd444ef4139348e551bb4beb343079b0e9f316bacccc8355ee27078abf362158f0c7fd24b840f5c6ecc38de7eadb995a7d41b312f253ac3f147ce4d6b";
+    const HASHED_TOKEN: &str =
+        "k-CL9sJ78p0n6TSnQ6z1-zfQo4Qwb8Lxl1ULQ1qwWSbRrgfbvxHd3m6t42jd46EgvJOsbuW_RCoBRXljmkoZ8g";
```

Comment:
Something is not right looking at this test. What we want to give to the user is `apiv3_<base64 of 512 bits of randomness>` and what we want the server to calculate for a match is `sha512(apiv3_<base64 of 512 bits of randomness>)`.

The value of `TOKEN` looks ok, but the `HASHED_TOKEN` does not. Eg, with this:
```
const TOKEN: &str = "apiv3_YpmcqSLB2wa5VkNbtIGvVe4BHVa4LcrhnhmTT9sEUNOpS96v_rdRR0P-81JV2xQGQ_3N3fd61xZYQc6ZTTHA2Q";
```

we should have:
```
// echo -n "apiv3_YpmcqSLB2wa5VkNbtIGvVe4BHVa4LcrhnhmTT9sEUNOpS96v_rdRR0P-81JV2xQGQ_3N3fd61xZYQc6ZTTHA2Q" | sha512sum
const HASHED_TOKEN: &str = "93e08bf6c27bf29d27e934a743acf5fb37d0a384306fc2f197550b435ab05926d1ae07dbbf11ddde6eade368dde3a120bc93ac6ee5bf442a014579639a4a19f2"
```

I'm not sure where the `"k-CL9sJ78p0n6TSnQ6z1-zfQo4Qwb8Lxl1ULQ1qwWSbRrgfbvxHd3m6t42jd46EgvJOsbuW_RCoBRXljmkoZ8g"` came from, but it seems we are double encoding?

Reply:
Ah yeah looks like it. I just need to convert the sha512 to a string directly. Should be an easy fix